src/viktrwlayer.c

```diff
@@ -10274,10 +10285,24 @@ static gboolean tool_edit_track_key_press ( VikTrwLayer *vtl, GdkEventKey *event
     update_statusbar ( vtl );
     vik_layer_emit_update ( VIK_LAYER(vtl) );
     return TRUE;
+  } else if ( event->keyval == GDK_Shift_L || event->keyval == GDK_Shift_R ) {
+    GdkWindow *gdkw = gtk_widget_get_window(GTK_WIDGET(vvp));
+    gdk_window_set_cursor ( gdkw, vtl->crosshair_cursor );
+    return TRUE;
   }
+
   return FALSE;
 }
 
+static gboolean tool_edit_track_key_release ( VikTrwLayer *vtl, GdkEventKey *event, VikViewport *vvp )
+{
+  if ( event->keyval == GDK_Shift_L || event->keyval == GDK_Shift_R ) {
+    VikWindow *vw = VIK_WINDOW(VIK_GTK_WINDOW_FROM_LAYER(vtl));
+    // this resets to standard tool cursor
+    vik_window_clear_busy_cursor ( vw );
+  }
+}
+
```

Comment:
This tool_edit_track_key_release () should be returning a value

Reply:
Sorry for that.  I've fixed it and updated the pull request.  Thanks.